modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/list-type-definitions/grant-request-type.list-type-entries.json

```
@@ -0,0 +1,26 @@
+[
+	{
+		"key": "blank",
+		"name": " ",
+		"name_i18n": {
+			"en-US": " "
```

Please substitute the blanck spaces for some content, this breaks the site building haha
```suggestion
		"name": "No Grant",
		"name_i18n": {
			"en-US": "No Grant"
```